tests/test_pim.py

```diff
@@ -56,8 +96,8 @@ def test_pim_subscription(fake_pim):
     logger.debug('test_pim_subscription')
     pim = fake_pim
     cb = Mock()
-    pim.subscribe(cb, event_type=pim.SUB_STATE, run=False)
-    pim.state.sim_put(2)
+    pim.state.subscribe(cb, event_type=pim.state.SUB_STATE, run=False)
+    pim.state.state.sim_put(2)
```

Comment:
I can't remember if there's a good reason to keep these top-level devices subscribeable. Maybe we were using that in `lightpath`? We should revisit in a later PR and do them uniformly if it's needed.

Reply:
Yeah, we could revisit that for lightpath if necessary.